### ISSUE TITLE
hi brother, when i'm reading the paper, i have some questions about the test acc on clothing1m which you presented in table2 

### ISSUE THREAD

**Issue by LanXiaoPang613** [jangsoohyuk/Robust-Federated-Learning-with-Noisy-Labels#3]
hi brother, 
      I must say it is a excellent work, but when i'm reading the paper, i have some questions about the test acc on clothing1m which you presented in table2. You gave the C.L. and F.L results of “ours”, do you mean the c.l. is the result that only uses one model to train the whole clothing1m dataset for several epochs? In other words, the F.L. means the result that uses the method you shown in algorithm1  to train?

**Comment by LanXiaoPang613** [comment]
Besides，when I reproduce the results in the cifar-10 dataset, the final test acc is only 69.33%**"the log is Round 999
train_acc: 55.666000 - train_loss: 6.664808 - test_acc: 69.330000 - test_loss: 3.884204
time : 88858.61390686035"** the hyper-parameters is the same as the paper, which is shown as follows:
![image](https://user-images.githubusercontent.com/35187545/230364950-23cfe4a5-22a8-4b47-b52c-114d8d4fcb50.png)
Could you see what's wrong?

**Comment by jangsoohyuk** [comment]
Thank you for reaching out and showing interest! However, as indicated in the repository, it is an "unofficial" repo., and I am not the author of the paper😢. Also, I created this repo when I was a college student for my own study purpose (it's been a long time since I wrote this code...😢😢). So I may not be able to answer your questions or the details about the experimental settings of the paper. I hope you understand. Thank you!

**Comment by jangsoohyuk** [comment]
btw, it may not be soon, but someday if I have time, I will try to reread the paper and reorganize the code. Thank you! 😊

**Comment by LanXiaoPang613** [comment]
hhh, it's ok, i have found the problem. I must say this work is very useful for us.

**Comment by LanXiaoPang613** [comment]
hhh, it's ok, i have found the problem. I must say this work is very useful for us.
